spotify_player/src/cli/commands.rs

```diff
@@ -128,3 +128,53 @@ pub fn init_like_command() -> Command {
 pub fn init_authenticate_command() -> Command {
     Command::new("authenticate").about("Authenticate the application")
 }
+
+pub fn init_playlist_subcommand() -> Command {
+    Command::new("playlist")
+        .about("Playlist editing")
+        .subcommand_required(true)
+        .subcommand(Command::new("new").about("Create a new playlist")
+            .arg(Arg::new("name")
+                .value_parser(clap::builder::NonEmptyStringValueParser::new()))
+            .arg(Arg::new("description")
+                .value_parser(clap::builder::NonEmptyStringValueParser::new())
+                .required(false))
+            .arg(Arg::new("public")
+                .short('p')
+                .long("public")
+                .action(clap::ArgAction::SetTrue)
+                .help("Sets the playlist to public"))
+            .arg(Arg::new("collab")
+                .short('c')
+                .long("collab")
+                .action(clap::ArgAction::SetTrue)
+                .help("Sets the playlist to collaborative"))
+            )
+        .subcommand(Command::new("delete").about("Delete a playlist")
+            .arg(Arg::new("id")
+                .value_parser(clap::builder::NonEmptyStringValueParser::new())))
+        .subcommand(Command::new("import").about("Imports all songs from a playlist into another playlist.")
+            .arg(Arg::new("from")
+                .value_parser(clap::builder::NonEmptyStringValueParser::new()))
+            .arg(Arg::new("to")
+                .value_parser(clap::builder::NonEmptyStringValueParser::new()))
+            .arg(Arg::new("delete")
+                .short('d')
+                .long("delete")
+                .action(clap::ArgAction::SetTrue)
+                .help("Deletes any tracks that were deleted by imported playlist since last import."))
+            .after_help("Copies all tracks into a playlist. If imported again, it adds new tracks and outputs information on deleted tracks since last import."))
```

Comment:
```suggestion
                .help("Deletes any previously imported tracks that are no longer in the imported playlist since last import."))
            .after_help("Import data for each playlist is stored inside the application's cache folder. If imported again, the command only adds new tracks since last import."))
```